src/main/java/io/stargate/sgv2/jsonapi/api/model/command/impl/CreateCollectionCommand.java

```diff
@@ -95,6 +95,20 @@ public record VectorSearchConfig(
             @JsonProperty("metric")
             @JsonAlias("function") // old name
             String metric,
+        @Nullable
+            @Pattern(
+                regexp =
+                    "(openai-v3-large|openai-v3-small|ada002|gecko|nv-qa-4|cohere-v3|bert|other)",
```

Comment:
Should these also be in alphabetic order (except for "other" that should be last I think)?